docs/systemd-usage.md

```diff
@@ -26,3 +26,12 @@ The procedure is as follows:
    # systemctl enable icyci@linux-kernel-stable
    # systemctl start icyci@linux-kernel-stable
    ```
+
+## Running as user service
+
+  * `mkdir -p $HOME/.config/systemd/user $HOME/.config/icyci`
+  * `ln -sr systemd-user/icyci@.service $HOME/.config/systemd/user/icyci@$CI_NAME.service
```

Comment:
I think it'd be clearer if you continue to use the `linux-kernel-stable` instance name for this example instead of `$CI_NAME`. If you want to keep `$CI_NAME`, then please add a note that it should be replaced with the name for the icyci instance.

Reply:
Also, is the instance name required for the user `.service` file? With regular (non-user) services we don't need it. I would have expected that the `EnvironmentFile=%h/.config/icyci/%i.conf` %i substitution would have been enough here too.